.travis.yml

```diff
@@ -206,40 +209,57 @@ base_osx_config: &base_osx_config
     brew:
       packages:
       - openssl
-  before_install:
-    - curl -L https://github.com/stedolan/jq/releases/download/jq-1.5/jq-osx-amd64 -o /usr/local/bin/jq
-    - chmod 755 /usr/local/bin/jq
-    - ./build-support/bin/install_aws_cli_for_ci.sh
-    - ./build-support/bin/install_python_for_ci.sh "${PYENV_PY36_VERSION}"
```

Comment:
I found this another confusing thing. As I understand it:
1. Having Py36 available on the PATH on all OSX shards is a feature waiting for a customer.
2, Having py27 available onthe PATH on all OSX shards is a needed feature (for build-support/bin/check_pants_pex_abi.py).

As such the old setup was installing an un-needed thing - it was noise when trying to figure out interpreter selection issues.

I sucked up the spirit of 1 by explicitly ensuring we have a sane 2.7 and 3.6+ on each OSX shard. We still don't need the 3.6+ yet though.

Generally I think it would be wise to avoid adding features to CI that we don't use. This beast is absurdly complex and it's hard enough to debug without unused bits floating about. Its nice to be able to assume every thing I'm reading is required and intentional.

Reply:
Okay. Contrary to earlier request, I'm fine with removing Py36 from the Py2 shards then.

I was planning on porting some of our scripts in between 1.15.x and 1.17.x, but given the tepid response to modernizing `check_pants_pex_abi.py` and `isort.py (which was warranted) and lots going on with school, gave up on this initiative. I'm not going to work on it the next few weeks, and by then we won't even have Python 2 shards.